lib/reporter.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+function installCommand(type) {
+  switch (type) {
+  case 'npm':
+    return 'npm install';
+  case 'npm-shrinkwrap':
+    return 'rm -rf node_modules/ && npm install';
```

Comment:
Changing that once dependency seems tricky to explain- and this message prints for a display of what may be many dependency errors.

This is aggressive, but it seems appropriate for a start.